airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/CtidUtils.java

```diff
@@ -104,8 +111,18 @@ private static List<ConfiguredAirbyteStream> identifyNewlyAddedStreams(final Con
         .collect(Collectors.toList());
   }
 
+  public static List<ConfiguredAirbyteStream> getStreamsFromStreamPairs(final ConfiguredAirbyteCatalog catalog,
+                                                                        final Set<AirbyteStreamNameNamespacePair> streamPairs) {
+
+    return catalog.getStreams().stream()
+        .filter(stream -> streamPairs.contains(AirbyteStreamNameNamespacePair.fromAirbyteStream(stream.getStream())))
+        .map(Jsons::clone)
+        .collect(Collectors.toList());
+  }
+
   public record StreamsCategorised(CtidStreams ctidStreams,
-                                   XminStreams xminStreams) {
+                                   XminStreams xminStreams,
+                                   StandardStreams standardStreams) {
 
```

Comment:
No problem but just wondering if code can reflect the fact that a sync is always ctid → a single type.
i.e there cannot be a sync with some streams standard and some xmin (and later on some CDC)